rstest_macros/src/render/fixture.rs

```diff
@@ -72,14 +75,33 @@ pub(crate) fn render(fixture: ItemFn, info: FixtureInfo) -> TokenStream {
         default_output = wrap_return_type_as_static_ref(default_output);
     }
 
+    let awaits = fixture
+        .sig
+        .inputs
+        .iter()
+        .filter(|a| {
+            a.maybe_ident()
+                .map(|i| info.await_args.contains(i))
+                .unwrap_or(false)
+        })
+        .map(|a| {
+            let ident = a
+                .maybe_ident()
+                .expect("Found a future argument without ident");
+            quote! {
+                let #ident = #ident.await;
+            }
+        });
+
     quote! {
         #[allow(non_camel_case_types)]
         #visibility struct #name {}
 
         impl #name {
             #(#orig_attrs)*
             #[allow(unused_mut)]
-            pub #asyncness fn get #generics (#orig_args) #output #where_clause {
```

Comment:
No particular issue. It's just that I changed this API, and when I saw the "pub" I thought "oh, maybe I can't change it after all, it'll affect a lot of code", but actually the function wasn't used anywhere so I just removed the pub. I thought it would be better to add pub if I need it than to put it by default.